fusillade/clouddirectory.py

```diff
@@ -678,6 +684,15 @@ def get_attributes(self, attributes: typing.List[str]):
             attrs[attr['Key']['Name']] = attr['Value'].popitem()[1]   # noqa
         return attrs
 
+    @staticmethod
+    def _verify_statement(statement):
```

Comment:
Consider adding a docstring explaining the level of statement verification.

For instance, `iam.simulate_custom_policy` uses fake data for `ActionNames` and `ResourceArns`. Does this indicate that the statement is only syntactically verified?

Reply:
adding documentation